tests/models/auto/test_tokenization_auto.py

```diff
@@ -314,12 +314,25 @@ def test_from_pretrained_dynamic_tokenizer(self):
 
         tokenizer = AutoTokenizer.from_pretrained("hf-internal-testing/test_dynamic_tokenizer", trust_remote_code=True)
         self.assertTrue(tokenizer.special_attribute_present)
+
+        # Test the dynamic module is loaded only once.
+        reloaded_tokenizer = AutoTokenizer.from_pretrained(
+            "hf-internal-testing/test_dynamic_tokenizer", trust_remote_code=True
+        )
+        self.assertIs(tokenizer.__class__, reloaded_tokenizer.__class__)
+        self.assertTrue(reloaded_tokenizer.special_attribute_present)
```

Comment:
no need to check `self.assertTrue(reloaded_tokenizer.special_attribute_present)`